src/master-playlist-controller.js

```diff
@@ -446,6 +446,22 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     this.audioPlaylistLoader_.start();
   }
 
+
+  /**
+   * Re-tune playback quality level for the current player
+   * conditions. This method may perform destructive actions, like
+   * removing already buffered content, to readjust the currently
+   * active playlist quickly.
+   */
+  fastQualityChange_() {
```

Comment:
If this is being used outside of master-playlist-controller (i.e., in videojs-contrib-hls), should we remove the underscore that identifies it as private?